get-started.md

```diff
@@ -1,5 +1,69 @@
 ---
 layout: get-started
-title: Get Started
+title: Kogito - Get Started
 permalink: /get-started/
 ---
+
+<div class="grid__item width-4-12 hide-mobile toc" markdown="1">
+## Table of Contents
+### Getting Started
+
+ - [Kogito business automation overview](https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-automation)
+ - [Creating and running your first Kogito services](https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-creating-running)
+ - [Example applications with Kogito services](https://docs.jboss.org/kogito/release/latest/html_single/#ref_kogito-app-examples_kogito-creating-running)
+ - [Deploying Kogito services on OpenShift](https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-deploying-on-openshift)
+ - [Travel agency tutorial for Kogito services on OpenShift](https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-travel-agency)
+
+### Guides
+
+ - [Get More Guides]({{site.baseurl}}/guides)
+
+</div>
+<div class="grid__item width-8-12 width-12-12-m gs-content">
+<div class="guide-item" markdown="1">
+## Kogito business automation overview
+
+Learn what is Kogito and when it can become an essential core of your domain-specific cloud-native set of services.
+
+<a href="https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-automation" class="button-cta secondary">READ THE GUIDE</a>
+</div>
+
+<div class="guide-item" markdown="1">
+## Creating and running your first Kogito services
+
+As a developer of business processes and rules, you can use Kogito business automation to build cloud-native applications that adapt to your business domain and tooling.
+
+This guide covers:
+
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-vscode-extension">Enabling the Kogito VSCode extension for BPMN and DMN modeling</a>
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-creating-project">Creating a new Kogito service project</a>
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-designing-app">Designing the application logic for a Kogito service using DMN and BPMN</a>
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-running-app">Running a Kogito service</a>
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#proc_kogito-testing-app">Testing a Kogito service</a>
+* <a href="https://docs.jboss.org/kogito/release/latest/html_single/#ref_kogito-app-examples_kogito-creating-running">Example applications with Kogito services</a>
+
+<a href="https://docs.jboss.org/kogito/release/latest/html_single/#chap_kogito-creating-running" class="button-cta secondary">START CODING</a>
```

Comment:
Is this button correct? I mean, it points to already seen topics like Enabling kogito vscode extension.

Reply:
Yes it points to the whole section "Creating and running your first Kogito services" where all the topics above are covered.

Reply:
alright.